src/common/models/CondaSpecification.ts

```diff
@@ -6,4 +6,5 @@ export type CondaSpecification = {
   dependencies: (string | CondaSpecificationPip)[];
   variables: Record<string, string>;
   prefix?: string | null;
+  lockfile?: any | null;
```

Comment:
I don't want to rely on any particular format here since it might change upstream.